media/js/plugincheck/check-plugins.js

```diff
@@ -157,13 +159,13 @@ $(function() {
     }
 
     // show main download button to non Fx traffic
-    if(!isFirefox() && !isLikeFirefox() ) {
+    if((!client.isFirefox && !client.isLikeFirefox) || client.isFirefoxiOS) {
```

Comment:
Why would we want to show the download button to users already on Firefox for iOS?